packages/react-components/lib/header-bar.stories.tsx

```diff
@@ -14,6 +14,16 @@ export default {
   component: HeaderBar,
 } as Meta;
 
+const StyledTab = styled((props: TabProps) => <Tab {...props} />)(({ theme }) => ({
+  color: 'rgba(255, 255, 255, 0.7)',
+  '&.Mui-selected': {
+    color: theme.palette.text.primary,
+  },
+  '&.Mui-focusVisible': {
+    backgroundColor: 'rgba(100, 95, 228, 0.32)',
+  },
+}));
+
```

Comment:
This is a copy of the `StyledTab` in `dashboard/appbar`. Without it, the Tab component would look weird. I think I will create a separate component for the StyledTab and remove the existing ant-tab since it is not in use.